yarn.lock

```diff
@@ -14008,6 +14008,11 @@ tinycolor2@^1.4.1:
   resolved "https://registry.yarnpkg.com/tinycolor2/-/tinycolor2-1.4.1.tgz#f4fad333447bc0b07d4dc8e9209d8f39a8ac77e8"
   integrity sha1-9PrTM0R7wLB9TcjpIJ2POaisd+g=
 
+tinycolor2@^1.4.2:
```

Comment:
Can you deduplicate the tinycolor resolutions so that just the most recent one is used?

Reply:
Done in a1bc3e9